nextcloud/android
# Feature - Add More MDM

**How to Test?**

1. Modify the relevant settings in setup.xml, and ensure the app responds appropriately to the updated conditions.
2. Verify if conditions. It must behave as expected.
3. Test both MDM-related variables and legacy variables in combination to confirm compatibility.



**Disable Multi Account -- User can't add account. New accounts can’t be added from the Manage Account section.**

<img src="https://github.com/user-attachments/assets/6fc02751-3a42-4aea-88fb-27333dd89111" alt="Screenshot" width="400"/>

<img src="https://github.com/user-attachments/assets/a68231a9-86b4-4222-95c6-abe3bc5c6741" alt="Screenshot" width="400"/>

**Disable Sharing -- Removes share button from action list**

<img src="https://github.com/user-attachments/assets/f5d0add7-7c20-42ec-b3b2-e70ad1fff9e3" alt="Screenshot" width="400"/>

<img src="https://github.com/user-attachments/assets/046a139b-76a7-4043-8f7b-6513265d5a3c" alt="Screenshot" width="400"/>

**Disable More External Site  -- Removes external sites from drawer menu**

<img src="https://github.com/user-attachments/assets/50b959dd-6503-49bc-9ec5-08c3ce0b47b3" alt="Screenshot" width="400"/>

**Disable Intro  -- Don't launch FirstRunActivity only if user not logged in before**


https://github.com/user-attachments/assets/327c6623-f451-4d6a-84d0-484a8f4c6d12
